src/pages/sites-and-applications.tsx

```diff
@@ -359,20 +362,31 @@ export default SitesAndApplications
 
 SitesAndApplications.getLayout = withDefaultLayout
 
-export async function getStaticProps() {
-  const collections: CollectionRecords = (await query.Collection.findMany({
-    query: 'id title bookmarks { id url label }',
-    where: {
-      showInSitesApps: {
-        equals: true,
+export async function getServerSideProps() {
+  const { error: cmsCollectionsError, data: cmsCollections } =
+    await client.query({
+      query: GET_KEYSTONE_COLLECTIONS,
+      context: {
+        clientName: 'cms',
       },
+      fetchPolicy: 'no-cache',
+    })
+
+  const collections = cmsCollections?.collections as CollectionRecords
+
+  const { error: cmsBookmarksError, data: cmsBookmarks } = await client.query({
+    query: GET_KEYSTONE_BOOKMARKS,
+    context: {
+      clientName: 'cms',
```

Comment:
Should this also have `fetchPolicy: 'no-cache'`?

Reply:
Yeah I believe so. I also meant to remove the error field as well, since Next handles that. I'll update it!